ansible/roles/screenly/files/uzbl-config

```diff
@@ -1,2 +1,6 @@
 set show_status = 0
-set ssl_verify = 1
+set ssl_verify  = 1
```

Comment:
@rusko124 I think this need a way to control this variable since we need to be able to support self-signed certificates.

Reply:
@vpetersson  I checked in the settings we have the use_ssl parameter.
And also we have this code
```
uzbl_rc = 'set ssl_verify = {}\n'.format('1' if settings['verify_ssl'] else '0')
with open(HOME + UZBLRC) as f:  # load uzbl.rc
    uzbl_rc = f.read() + uzbl_rc
```
in server.py
So we can just delete this set line from config.

Reply:
Got it!